### PR TITLE
[DE] Update `Tournaments/Official support`, `People/Users with unique titles` and `Community/osu! community meetings`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Tasks

- [x] Check if #10047 requires changes (and implement)
- [x] Check if #10041 requires changes (and implement)
- [x] Check if #10050 requires changes (and implement)
- [x] Check if #10051 requires changes (and implement)

## Notes

~~*Note: The information listed here may not be usable yet for a review, since the changes still need to be applied locally and/or a self-review is underway. I will leave a dedicated comment as soon as my work is finished.*~~

- `bracket`: Normally, I would have translated this with `Tunierbaum`. In the case you can find in the article (line 84), this didn't make sense to me, so I used `Gruppe(nphase)`.
- `bracket match`: I'm not entirely sure if this has some special meaning compared to just `match`. I translated it to `Entscheidungsspiel`.
- hinreichend teilgenommen: `reasonably` can be translated as "behaved well" and "put in enough effort". This rule specifically refers to people that qualify together with a team for the badge prizes. Receiving badges without playing enough matches is forbidden.
- Alternatives for line 189: `Präsenz`, `Sichtbarkeit bei den Spielern`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/osu!_community_meetings/de.md [hunk 1]
```diff
@@ -52,6 +47,7 @@ Das erste osu! Community Meeting fand am 19. September 2021 statt. Alle Treffen
 | 14 | [3. April 2022](https://youtu.be/UWT18LaoeKw) | [Sitzungsnotizen](https://docs.google.com/document/d/1LzKpXwIKxcpYgEAK4zdEIVuMNJckoo9SWN-UoAvOto8) | Auswertung der Scoring-Umfrageergebnisse, diverse Fragen aus der Community |
 | 15 | [12. Februar 2023](https://youtu.be/idmI03A8jR8) | [Sitzungsnotizen](https://docs.google.com/document/d/13cMCrQN4vvaQFA59BmFHk6D7PH9fwT2ANmhkm5cmzxE) | Spielmechanik und Balance in osu!(lazer) |
 | 16 | [7. Juli 2023](https://youtu.be/bocgaVISSfg) | [Zusammenfassender Newsbeitrag](https://osu.ppy.sh/home/news/2023-07-11-community-meeting-recap) | Finalisierung des Punktesystems in osu!(lazer) |
+| 17 | [21. August 2023](https://youtu.be/O2j00E6AVW8) | [Übersichtsvideo](https://youtu.be/crkT0aaowKQ) | Verlagerung der Entwicklung auf die Fertigstellung von Ranglistenspielen |
```
Comment (respektive) [inline wiki/Community/osu!_community_meetings/de.md:50]:
```suggestion
| 17 | [21. August 2023](https://youtu.be/O2j00E6AVW8) | [Zusammenfassendes Video](https://youtu.be/crkT0aaowKQ) | Verlagerung des Entwicklungsschwerpunkts auf die Fertigstellung von Ranglistenspielen |
```
Mabye works a bit better

File: wiki/People/Users_with_unique_titles/de.md [hunk 2]
```diff
@@ -269,6 +264,8 @@ Diese Storyboarder haben den Titel **Elite Storyboarder** dafür überreicht bek
 
 ## Auf das Spiel bezogen
 
+*Siehe auch: [Completionists](/wiki/People/Completionists)*
+
 Diesen Spielern wurden ihre entsprechenden Titel entweder durch eine verrückte Errungenschaft im Spiel oder durch Abschluss einer spezifischen  Herausforderung vergeben.
```
Comment (respektive) [inline wiki/People/Users_with_unique_titles/de.md:269]:
```suggestion
Diesen Spielern wurden ihre entsprechenden Titel entweder durch eine verrückte Errungenschaft im Spiel oder durch Abschluss einer spezifischen Herausforderung vergeben.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/People/Users_with_unique_titles/de.md [hunk 3]
```diff
@@ -279,7 +276,9 @@ Diesen Spielern wurden ihre entsprechenden Titel entweder durch eine verrückte
...
 | ::{ flag=NL }:: [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | Wurde durch ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) Ende 2008 überreicht für das Erreichen des Rangs #104 in einer Woche, als eine Referenz auf [*The World Ends with You*](https://de.wikipedia.org/wiki/The_World_Ends_with_You) <!-- source: https://osu.ppy.sh/community/forums/posts/60664 (Shirotora_old's forum signature) --> |
 | ::{ flag=US }:: [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Spielte alle gerankten osu!-Beatmaps ab dem 27. April 2019 |
-| ::{ flag=DE }:: [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter | Gewann [The Hunt for the Girl in the Forest](https://osu.ppy.sh/community/forums/topics/662279) |
+| ::{ flag=DE }:: [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter | War die [erste Person, die die Medaille *The Girl in the Forest* freischaltete](https://osu.ppy.sh/community/forums/topics/662279) |
+| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) | Medal Hunter | War die [erste Person, die die Medaille *Internment* freischaltete](https://twitter.com/osugame/status/1696007320663359800?s=20) |
```
Comment (agatemosu) [inline wiki/People/Users_with_unique_titles/de.md:280]: nothing but
```suggestion
| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) | Medal Hunter | War die [erste Person, die die Medaille *Internment* freischaltete](https://twitter.com/osugame/status/1696007320663359800) |
```

File: wiki/Tournaments/Official_support/de.md [hunk 4]
```diff
@@ -81,9 +79,9 @@ Zusätzlich sollten alle Werbematerialien oder andere Dienstleistungen, die im Z
...
-- Ein engagierter Schiedsrichter muss bei jedem Match präsent sein. Spieler können nicht selbst Schiedsrichter sein.
+- Ein fest zugeordneter Schiedsrichter muss bei jedem Match anwesend sein oder innerhalb eines minimalen Zeitraums zur Verfügung stehen (2 bis 5 Minuten), wenn eine entsprechende Genehmigung für automatisierte Software eingeholt wurde und zur Unterstützung eingesetzt wird. Spieler können nicht selbst Schiedsrichter sein.
   - Der Einsatz von Bots ist für die Qualifizierungsphase gestattet, sofern dies von einem Mitarbeiter beaufsichtigt wird.
-- Asynchrone Spiele sind unter keinen Umständen erlaubt. Alle teilnehmenden Teams müssen zur geplanten Spielzeit anwesend sein.
+- Jedes Spiel innerhalb derselben Gruppen- oder Qualifikationsrunde sollte in einem einheitlichen Format sein. In einem Turnier, das zum Beispiel eine Qualifikationsphase und ein Doppeleliminierungsverfahren verwendet, sollten alle Entscheidungsspiele entweder synchron oder asynchron gespielt werden, aber nicht beides.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/Official_support/de.md:84]: - `bracket`: Normally, I would have translated this with `Tunierbaum`. In the case above, this didn't make sense to me, so I used `Gruppe(nphase/runde)`.
- `bracket match`: I'm not entirely sure if this has some special meaning compared to just `match`. I translated it to `Entscheidungsspiel`.

File: wiki/Tournaments/Official_support/de.md [hunk 5]
```diff
@@ -102,14 +111,17 @@ In Bezug auf die Rolle des Personals gelten folgende Regeln:
...
+Von Teilnehmern in offiziell unterstützten Turnieren wird erwartet, dass sie sich zu jedem Zeitpunkt an die [osu! Community-Regeln](/wiki/Rules) halten, unabhängig von ihrem Rang, ihren Errungenschaften oder anderen Leistungen. Von ihnen wird ebenfalls erwartet, dass sie alle Freiwilligen, die an der Ausrichtung eines Turniers beteiligt sind, mit Respekt behandeln und nicht absichtlich versuchen, ihre Zeit zu verschwenden oder anderweitig mit ihren Aktionen die Durchführung eines Turniers übermäßig zu erschweren.
+
+Des Weiteren wird von den Spielern erwartet, dass sie an einem bestimmten Turnier *hinreichend teilgenommen* haben, um sich für etwaige Preise zu qualifizieren — sich einfach als Mitglied eines Siegerteams zu qualifizieren ist nicht ausreichend.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/Official_support/de.md:124]: **hinreichend teilgenommen**: `reasonably` can be translated as "behaved well" and "put in enough effort". This rule specifically refers to people that qualify together with a team for the badge prizes. Receiving badges without playing enough matches is forbidden.

See [this discussion on Discord](https://discord.com/channels/188630481301012481/867773019015610389/1152574080319492158).

File: wiki/Tournaments/Official_support/de.md [hunk 6]
```diff
@@ -164,23 +176,35 @@ Abzeichenbilder, die diese Standards nicht erfüllen, werden mit angegebener Beg
...
 
 - [Corsace Open](https://osu.ppy.sh/community/forums/topics/1607468)
 - [Expert Global Taiko Showdown](/wiki/Tournaments/GTS/EGTS_2022)
 - [Springtime osu!mania Free-for-all Tournament](/wiki/Tournaments/SOFT/6)
 
-### Unterstützung von Bannern im Hauptmenü beantragen
+### Werbung durch Banner im Spiel und in Newsbeiträgen beantragen
 
-Langjährige Turniere von der Community können sich für Banner im Hauptmenü des Spiels qualifizieren. Es liegt an dem Ermessen des osu!-Teams zu entscheiden, ob dein Turnier im Spiel beworben werden kann, da es viele Faktoren zu beachten gibt. Die minimalen Anforderungen für die Beantragung von Werbung im Spiel sind wie folgt:
+Langjährige Turniere von der Community können sich für zusätzliche Unterstützung qualifizieren, um ihre Reichweite in der Spielergemeinschaft zu erhöhen, sprich Banner im Hauptmenü des Spiels und Öffentlichkeitsarbeit durch Newsbeiträge auf der osu!-Webseite.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/Official_support/de.md:189]: Alternatives for the first phrase: `Präsenz`, `Sichtbarkeit bei den Spielern`

**Review by The-Last-Cookie — COMMENTED** [review]
The changes are now implemented and can be reviewed.

I've added the comments from the PR's description directly under the corresponding lines to make the review easier.

**Review by respektive — APPROVED** [review]
:+1:

**Review by TicClick — APPROVED** [review]

**Pull request merged**